src/scene/materials/standard-material-options-builder.js

```diff
@@ -347,8 +351,17 @@ class StandardMaterialOptionsBuilder {
                 if (stdMat[uname] === 0 && !hasUv0) allow = false;
                 if (stdMat[uname] === 1 && !hasUv1) allow = false;
                 if (allow) {
+
+                    const mapId = stdMat[mname].id;
+                    let identifier = textureMap[mapId];
+                    if (identifier === undefined) {
+                        textureMap[mapId] = textureMappingCounter.value;
```

Comment:
I'd change the names to make it clearer. `textureMap` usually means a texture map.
textureMap -> uniqueTexturesMap
identifier -> uniqueTextureId